README.md

```diff
@@ -97,12 +97,12 @@ Then in a separate terminal run:
 npx wrangler pages dev .vercel/output/static --compatibility-flag=nodejs_compat
 ```
 
-### Install `@cloudflare/next-on-pages` and vercel (optional)
+### Install `@cloudflare/next-on-pages` and next-on-pages-vercel-cli (optional)
 
-To speed up local development (especially the refresh speed when running in watch mode) you can optionally choose to install `@cloudflare/next-on-pages` and `vercel` as dev dependencies of your project:
+To speed up local development (especially the refresh speed when running in watch mode) you can optionally choose to install `@cloudflare/next-on-pages` and `next-on-pages-vercel-cli` as dev dependencies of your project:
 
 ```sh
-npm install -D @cloudflare/next-on-pages vercel
+npm install -D @cloudflare/next-on-pages next-on-pages-vercel-cli
```

Comment:
I thought this was just a temporary thing to get it working in Pages' v2 build system? I wouldn't document it — especially if it's just for local development and unnecessary there.

Reply:
> Where is the fork? If not in the Cloudflare org, I think it should be open-source at least somewhere (your personal account?) so inquisitive folks can see what code we're running.

It's in the PR's description (I had the link wrong, I've just updated it 🙇) I hope that's enough

Reply:
yeah it's totally temporary, so I updated this because I don't want to keep the vercel part while its going to be totally inaccurate (but I kept things rather vague 😅)

I'll fix this thanks 🙂

Reply:
I'll just (temporarily) remove this whole section is that ok by you 😕